### DISCUSSION TITLE
API and Process Mapping

### DISCUSSION THREAD

**Discussion by dasunhegoda** [spdci/.github#8] [Ideas]
Hey everyone,

Have a suggestion - do you think it would be helpful to create a Process-to-API mapping? This would help us better understand which processes are utilized in which APIs. 

Let me know your thoughts!

**Comment by KomalMalhotra** [comment]
I agree that having a process and API mapping will be good to guide on how to use the API within a specific context.  we will have prepare a matrix for it
